### ISSUE TITLE
[Tracking Website Request] Add tracking support for the new MangaUpdates API

### ISSUE THREAD

**Issue by rRoler** [MALSync/MALSync#1130]
[MangaUpdates released their new API](https://www.mangaupdates.com/topic/f0ghpti/mangaupdates-api-released?page=1#post797121). 

https://api.mangaupdates.com/

**Comment by toprak** [comment]
any updates?

**Comment by lolamtisch** [comment]
Not highest priority ATM. Specially as we still need to find a good way to handle the mapping to mal ids.

**Comment by shashankx86** [comment]
Any ETA for this?

**Comment by lolamtisch** [comment]
Yes, no eta

**Comment by shashankx86** [comment]
> Yes, no eta

Do guys even plan of adding support for this?
I mean this issue was made 2y ago

**Comment by shashankx86** [comment]
any updates?

**Comment by d3lm4n** [comment]
Mal or any other tracker are very hit or miss with manga. Mangaupdates would've been really great.

**Comment by IkelAtomig** [comment]
It's been 2 years. It would be great if supported. Currently unavailable titles are tracked only locally and not synced to other devices. MangaUpdates would solve the issue.

**Comment by roakizi** [comment]
<img width="244" height="244" alt="Image" src="https://github.com/user-attachments/assets/0d5fb356-fdc9-4239-854b-31e4379bf705" />

There's no viable way to import my wishlist from mangaupdates to MAL, and I find the MAL manga lists to be, well, bad.

**Comment by GitBubblyWubbly** [comment]
> Not highest priority ATM. Specially as we still need to find a good way to handle the mapping to mal ids.

I think many people, myself included, would already be very happy with a first version that only adds Mangaupdates, without the ability to sync with MAL or the other trackers.
Especially because for most manga/manhwa/manhua Mangaupdates is the only existing tracker, which means that in most cases there's nothing to sync anyway.

So if it's possible to realise this feature sooner when the "mapping to mal ids"-functionality is left out of scope please consider doing so.

**Comment by shashankx86** [comment]
i agree with @GitBubblyWubbly Mangaupdates has every ga/hwa/hua/etc under the sky if something isnt there then it probably doesnt exists, not having this is such a loss. The only reason im with anilist cuz it works with mal sync, else i would have been on MU
